src/com/content-games/games.js

```diff
@@ -152,14 +156,28 @@ export default class ContentGames extends Component {
 				return <ContentItemBox node={r.node} user={props.user} path={props.path} noevent={props.noevent ? props.noevent : null} />;
 			});
 
+			while (Games.length % layout !== 0) {
+				Games.push(<ContentItemBox placeHolder={true} />);
+			}
+
 			if ( !props.nomore /*|| added >= 10*/ ){
 				LoadMore = <ContentMore onclick={this.fetchMore} />;
 			}
 
-			return(
+			const gridClass = '-columns-' + layout;
+
+			return (
 				<div class={cN(Class, props.class)}>
 					{props.children}
-					<div class='content-boxes'>
+					<GridSelector
+						defaultLayout={defaultLayout}
+						onChangeLayout={
+							(gridLayout) => {
+								this.setState({layout: gridLayout,});
```

Comment:
Please wrap member names in string quotes. i.e. `{'layout': ...}`. For consistency with JSON.